sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/implementation/models/TasksStateTasksDetails.java

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.ai.textanalytics.implementation.models;
+
+import com.azure.core.annotation.Immutable;
+
+/** The TasksStateTasksDetails model. */
+@Immutable
+public final class TasksStateTasksDetails extends TaskState {}
```

Comment:
nit: @jianghaolu @srnagar this is a pain point in code generation for a class with no body as it generates as `{}` which then fails Checkstyle

Reply:
😭

Reply:
https://github.com/Azure/autorest.java/issues/515
https://github.com/Azure/autorest.java/issues/823
https://github.com/Azure/autorest.java/issues/797